metadata-transformer/src/main/java/no/unit/nva/metadata/DcTerms.java

```diff
@@ -64,20 +65,30 @@
     VALID("valid");
 
     private static final String DCTERMS_PREFIX = "http://purl.org/dc/terms/";
+    public static final String DC = "dc.";
+    public static final String DCTERMS = "dcterms.";
+    public static final String EMPTY_STRING = "";
 
     private final String localName;
 
     DcTerms(String localName) {
         this.localName = localName;
     }
 
+    @Override
```

Comment:
As the "mappings" are consistent and not to be understood as extensible outside the dcterms namespace, I have chosen to create the IRI in this way.